cli/injector/main.go

```diff
@@ -348,6 +348,13 @@ func initPodWatch(resourceVersion string) (<-chan watch.Event, error) {
 		ResourceVersion:     resourceVersion,
 		AllowWatchBookmarks: true,
 	})
+	if err != nil {
+		return nil, err
+	}
+
+	if podWatcher == nil {
+		return nil, fmt.Errorf("unable to watch pod %s in namespace %s", disruptionArgs.TargetName, disruptionArgs.DisruptionNamespace)
```

Comment:
simple fix haha, nice!

Reply:
should we retry to watch the pod after this error? Can we?

Reply:
We do! The caller, `watchTargetandReinject` is already in a `backoff.RetryNotify` call, so we'll give a few attempts